File: Week6/sentimental-mario-more/mario.py
#promt input(:Height: ")
#if input != integer, re-prompt
# i = 1
#for row in rows
    #print " " * (Height - i)
    #print "#" * i
    #print " "
    #print "#" * i
Height = input("Height: ")
while (Height.isdigit() != True or int(Height) < 1 or int(Height) > 8):
    Height = input("Height: ")

Height = int(Height)
if Height == 1:
    print("#  #")
else:
    for i in range(1, (Height + 1), 1):
        print(" " *(Height - i) + "#" * i + "  " + "#" * i)

File: Week1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    //promt user to input height
    do
    {
        height = get_int("Height: ");
    }
    while(height > 8 || height < 1);
    // Generate number of space to print
        int spaceCount = height - 1;
    // Generate number of # to print
        int hashCount = height - spaceCount;

    //Generate the triangle
    for(int j = 0; j < height; j++ )
    {

        for(int i = 0; i < spaceCount; i++)
        {
            printf(" ");
        }

        for(int y = 0; y < hashCount; y++)
        {
            printf("#");
        }

        printf("  ");

        for(int z = 0; z < hashCount; z++)
        {
            printf("#");
        }

        printf("\n");

        spaceCount--;
        hashCount++;

    }



















    // write a space bar vertical row

    //write a loop to generate an inverted angle triangle
}